docs/community/old-setup/old-hostsystem.md

```diff
@@ -1,169 +0,0 @@
----
-title: Host system
-layout: default
-parent: Community setup
-grand_parent: Community
----
-
-# Host System
-
-## Operating System Configuration
-
-This page discusses software configuration issues commonly encountered when deploying FeatureBase. For hardware requirements or scaling, look at [Sizing FeatureBase](/docs/community/com-config/old-size-featurebase-database).
-
-FeatureBase's data storage splits the database into a potentially large number of files, which are open simultaneously. Most Unix-derived systems limit the number of simultaneous open files for a given process or user, and these limits are frequently small enough to be a problem. Similarly, there may be limits on the number of memory-mapped regions allowed in a process. These may need to be changed,
-either temporarily during initial development and testing, or as a machine-wide configuration change that persists across restarts.
-
-Throughout these instructions, commands given with a root prompt (`#`) are expected to require root privileges; you can run them in a root shell, or using `sudo`.
-
-### Linux Systems
-
-You can make temporary changes just to run the software once, or persistent changes which will survive reboots. We document both because the one-off changes are much easier to make, but in a production environment you will be happier making persistent changes.
-
-To temporarily increase the open file limit, you should be able to use `ulimit -n`. If you can't set the limit high enough, you will need root privileges. The `ulimit` program only affects the limits of the shell it's run in and child processes of that shell, so you can't use `sudo ulimit` to change the limit of your current shell; if you need to set a higher limit, launch a new shell using `sudo -s` or the equivalent, then change the ulimit, then use `su` to start a new shell running as your regular user but inheriting the higher limits. We do not recommend running FeatureBase with root privileges.
-
-To temporarily increase the allowable memory-mapped regions on a Linux system, you can use the `sysctl` command, or just the `proc` filesystem:
-
-```shell
-# sysctl -w vm.max_map_count=N
```

Comment:
why was this content no included in the new stuff?

Reply:
As above

Reply:
Again, this is something users might face to get featurebase running. It's a hint of something they may have to do, which would be crazy hard to find externally.

Reply:
done